xclim/testing/tests/test_precip.py

```diff
@@ -371,6 +371,13 @@ def test_last_snowfall(self):
         )
 
 
+class TestDaysWithSnow:
+    def test_simple(self, prsn_series):
+        prsn = prsn_series(np.arange(366), start="1999-07-01")
+        out = atmos.days_with_snow(prsn, low="10 kg m-2 s-1", high="20 kg m-2 s-1")
+        assert out[0] == 11
```

Comment:
The "ERA5/daily_surface_cancities_1990-1993.nc" test dataset has prsn series if you want to test with real data.

Reply:
Ok thanks, done.